### PR TITLE
Rocks db extractor

### PR DESCRIPTION
**Pull request by kasiaturbiarz**

### CODE REVIEW

File: src/Nethermind/Nethermind.Db.Rocks/Config/DbConfig.cs [hunk 1]
```diff
@@ -98,6 +98,36 @@ public class DbConfig : IDbConfig
         public ulong CanonicalHashTrieDbBlockCacheSize { get; set; } = 8.MB();
         public bool CanonicalHashTrieDbCacheIndexAndFilterBlocks { get; set; } = true;
         
+        public bool DataAssetsDbCacheIndexAndFilterBlocks { get; set; } = false;
```
Comment (tkstanczak) [inline src/Nethermind/Nethermind.Db.Rocks/Config/DbConfig.cs:101]: is this needed?

File: src/Nethermind/Nethermind.Db.Rocks/Config/IDbConfig.cs [hunk 2]
```diff
@@ -79,24 +79,53 @@ public interface IDbConfig : IConfig
...
         ulong EthRequestsDbBlockCacheSize { get; set; }
         bool EthRequestsDbCacheIndexAndFilterBlocks { get; set; }
-        
+
         ulong BloomDbWriteBufferSize { get; set; }
         uint BloomDbWriteBufferNumber { get; set; }
         ulong BloomDbBlockCacheSize { get; set; }
         bool BloomDbCacheIndexAndFilterBlocks { get; set; }
 
+        ulong DataAssetsDbBlockCacheSize { get; set; }
```
Comment (tkstanczak) [inline src/Nethermind/Nethermind.Db.Rocks/Config/IDbConfig.cs:93]: is this needed?

File: src/Nethermind/Nethermind.Db.Rocks/Config/DbConfig.cs (hunk 1 above)
Comment (kasiaturbiarz) [reply src/Nethermind/Nethermind.Db.Rocks/Config/DbConfig.cs:101]: yes. to properly get data. there was missing declaration in idbconfig interface - just added.

File: src/Nethermind/Nethermind.Db.Rocks/Config/IDbConfig.cs (hunk 2 above)
Comment (kasiaturbiarz) [reply src/Nethermind/Nethermind.Db.Rocks/Config/IDbConfig.cs:94]: yes. to properly get data

**Review by tkstanczak — APPROVED** [review]

**Pull request merged**